Competition2018/src/main/java/competition/subsystems/elevator/commands/ElevatorMaintainerCommand.java

```diff
@@ -26,7 +26,9 @@ public void initialize() {
 
     @Override
     public void execute() {
+        if(elevator.isCalibrated()) {
         double power = pid.calculate(elevator.getTargetHeight(), elevator.currentHeight());
         elevator.setPower(power);
+        }
```

Comment:
If the elevator isn't calibrated maybe you should throw an error or log that the elevator isn't calibrated.